.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.formatting.provider": "black"
```

Comment:
Could you remove this file from the commit history?

Reply:
You can add this file to your global gitignore.

Reply:
thanks, removed